SimplePolicy.UnitTests/UT_SimplePolicy.cs

```diff
@@ -290,6 +290,90 @@ public void TestMock_GenerateInvocationTransaction()
         }
 
+         [TestMethod]
+        public void TestVerifySizeLimits_FreeLessEq1024()
+        {
+            var txLowPriority = MockGenerateInvocationTransaction(new Fixed8(100000000 / 10000), 1024).Object; // 0.00001
+            txLowPriority.IsLowPriority.Should().Be(true);
+            txLowPriority.Size.Should().Be(1024); 
+            Settings.Default.MaxFreeTransactionSize.Should().Be(1024);
+            
+            uut.VerifySizeLimits(txLowPriority).Should().Be(true); // 1024 <= 1024
+        }
+
+
+        [TestMethod]
+        public void TestVerifySizeLimits_FreeGreater1024()
+        {
+            var txLowPriority = MockGenerateInvocationTransaction(new Fixed8(100000000 / 10000), 1025).Object; // 0.00001
+            txLowPriority.IsLowPriority.Should().Be(true);
+            txLowPriority.Size.Should().Be(1025); 
+            Settings.Default.MaxFreeTransactionSize.Should().Be(1024);
+            
+            uut.VerifySizeLimits(txLowPriority).Should().Be(false); // 1025 > 1024
+        }
+
+
+        [TestMethod]
+        public void TestVerifySizeLimits_HighPriorityLessEq1024()
+        {
+            var txHighPriority = MockGenerateInvocationTransaction(new Fixed8(100000000 / 1000), 1024).Object; // 0.0001
+            txHighPriority.IsLowPriority.Should().Be(false);
+            txHighPriority.Size.Should().Be(1024); 
+            Settings.Default.MaxFreeTransactionSize.Should().Be(1024);
+            
+            uut.VerifySizeLimits(txHighPriority).Should().Be(true); // 1024 <= 1024
+        }
+
+
+        [TestMethod]
+        public void TestVerifySizeLimits_HighPriorityGreater1024_1025()
+        {
+            var txHighPriority = MockGenerateInvocationTransaction(new Fixed8(100000000 / 1000), 1025).Object; // 0.0001
+            txHighPriority.IsLowPriority.Should().Be(false);
+            txHighPriority.Size.Should().Be(1025); 
+            Settings.Default.MaxFreeTransactionSize.Should().Be(1024);
+            Settings.Default.FeePerExtraByte.Should().Be(new Fixed8(100000000 / 100000)); // 0.000001 (1000 satoshi)
+            
+            uut.VerifySizeLimits(txHighPriority).Should().Be(true); // 1025 > 1024 (extra fee was 1 byte * 1000 satoshi = 0.000001)
+        }
+
+        [TestMethod]
+        public void TestVerifySizeLimits_HighPriorityGreater1024_1124()
+        {
+            var txHighPriority = MockGenerateInvocationTransaction(new Fixed8(100000000 / 1000), 1124).Object; // 0.0001
+            txHighPriority.IsLowPriority.Should().Be(false);
+            txHighPriority.Size.Should().Be(1124); 
+            Settings.Default.MaxFreeTransactionSize.Should().Be(1024);
+            Settings.Default.FeePerExtraByte.Should().Be(new Fixed8(100000000 / 100000)); // 0.000001 (1000 satoshi)
+            
+            uut.VerifySizeLimits(txHighPriority).Should().Be(true); // 1124 > 1024 (extra fee was 100 byte * 1000 satoshi = 0.0001)
+        }
+
+         [TestMethod]
```

Comment:
Indentation

Reply:
Let's fix this on lights branch... if we go in that direction. I had to go fast because discussion was getting crazy on these numbers ahahah